src/mmpars.c

```diff
@@ -5921,113 +5923,113 @@ vstring readInclude(const char *fileBuf, long fileBufOffset,
-      the parent file */
-  g_IncludeCall[g_includeCalls].current_includeSource = ""; /* (Currently) assigned
-      only if we may need it for a later Begin comparison */
-  g_IncludeCall[g_includeCalls].current_includeLength = *size; /* Length of the file
-      to be included (0 if the file was previously included) */
-
-  /* Create a fictitious entry for the "continuation" after the
-     main file, to make error message line searching easier */
+  // $[ $], Begin, of Skip file name for this inclusion
+  let(&g_IncludeCall[g_includeCalls].included_fn, inputFn);
+  // This is the starting character position of the included file w.r.t entire source buffer 
+  g_IncludeCall[g_includeCalls].current_offset = 0;
+  // The line number of the start of the included file (=1) or the
+  // continuation line of the parent file.
+  g_IncludeCall[g_includeCalls].current_line = 1;
+  // (Currently) assigned only if we may need it for a later Begin comparison
+  g_IncludeCall[g_includeCalls].current_includeSource = "";
+  // Length of the file to be included (0 if the file was previously included)
+  g_IncludeCall[g_includeCalls].current_includeLength = *size;
+
+  // Create a fictitious entry for the "continuation" after the
+  // main file, to make error message line searching easier.
   g_includeCalls++;
-  g_IncludeCall[g_includeCalls].pushOrPop = 1; /* 0 means start of included file,
-       1 means continuation of including file */
+  // 0 means start of included file, 1 means continuation of including file
+  g_IncludeCall[g_includeCalls].pushOrPop = 1;
   g_IncludeCall[g_includeCalls].source_fn = "";
-  /*let(&g_IncludeCall[g_includeCalls].source_fn, inputFn);*/ /* Leave empty;
-        there is no "continuation" file for the main file, so no need to assign
-        (it won't be used) */
+  // let(&g_IncludeCall[g_includeCalls].source_fn, inputFn); // Leave empty;
+  // there is no "continuation" file for the main file, so no need to assign
+  // (it won't be used).
   g_IncludeCall[g_includeCalls].included_fn = "";
-  /*let(&g_IncludeCall[g_includeCalls].included_fn, inputFn);*/ /* $[ $], Begin,
-      of Skip file name for this inclusion */
-  g_IncludeCall[g_includeCalls].current_line = -1; /* Ideally this should be
-      countLines(fileBuf), but since it's never used we don't bother to
-      call countLines(fileBuf) to save CPU time */
-  g_IncludeCall[g_includeCalls].current_includeSource = ""; /* (Currently) assigned
-      only if we may need it for a later Begin comparison */
-  g_IncludeCall[g_includeCalls].current_includeLength = 0; /* The "continuation"
-      of the main file is fictitious, so just set it to 0 length */
-
-  /* Recursively expand the source of an included file */
+  // let(&g_IncludeCall[g_includeCalls].included_fn, inputFn); // $[ $], Begin,
+  // of Skip file name for this inclusion.
+  // Ideally this should be countLines(fileBuf), but since it's never used we don't
```

Comment:
```suggestion
  // $[ $], Begin, of Skip file name for this inclusion.
  // let(&g_IncludeCall[g_includeCalls].included_fn, inputFn);

  // Ideally this should be countLines(fileBuf), but since it's never used we don't
```